2021-04-11 21:21:20,193 INFO    MainThread:30675 [wandb_setup.py:_flush():69] setting env: {}
2021-04-11 21:21:20,193 INFO    MainThread:30675 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-11 21:21:20,193 INFO    MainThread:30675 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210411_212120-3se8ouuc/logs/debug.log
2021-04-11 21:21:20,193 INFO    MainThread:30675 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210411_212120-3se8ouuc/logs/debug-internal.log
2021-04-11 21:21:20,194 INFO    MainThread:30675 [wandb_init.py:init():369] calling init triggers
2021-04-11 21:21:20,194 INFO    MainThread:30675 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-11 21:21:20,195 INFO    MainThread:30675 [wandb_init.py:init():418] starting backend
2021-04-11 21:21:20,195 INFO    MainThread:30675 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-11 21:21:20,217 INFO    MainThread:30675 [backend.py:ensure_launched():123] starting backend process...
2021-04-11 21:21:20,226 INFO    MainThread:30675 [backend.py:ensure_launched():128] started backend process with pid: 30688
2021-04-11 21:21:20,228 INFO    MainThread:30675 [wandb_init.py:init():423] backend started and connected
2021-04-11 21:21:20,231 INFO    MainThread:30675 [wandb_init.py:init():463] updated telemetry
2021-04-11 21:21:20,232 INFO    MainThread:30675 [wandb_init.py:init():482] communicating current version
2021-04-11 21:21:20,953 INFO    MainThread:30675 [wandb_init.py:init():487] got version response 
2021-04-11 21:21:20,953 INFO    MainThread:30675 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-11 21:21:21,151 INFO    MainThread:30675 [wandb_init.py:init():520] starting run threads in backend
2021-04-11 21:21:21,222 INFO    MainThread:30675 [wandb_run.py:_console_start():1524] atexit reg
2021-04-11 21:21:21,223 INFO    MainThread:30675 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-11 21:21:21,224 INFO    MainThread:30675 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-11 21:21:21,227 INFO    MainThread:30675 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-11 21:21:21,227 INFO    MainThread:30675 [wandb_init.py:init():544] run started, returning control to user process
2021-04-11 23:58:27,651 INFO    MainThread:30675 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-11 23:58:27,656 INFO    MainThread:30675 [wandb_run.py:_restore():1466] restore
2021-04-11 23:58:29,840 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 144589
  total_bytes: 144589
}

2021-04-11 23:58:30,285 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 144589
  total_bytes: 144589
}

2021-04-11 23:58:30,488 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 144589
  total_bytes: 146128
}

2021-04-11 23:58:30,673 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 145506
  total_bytes: 146128
}

2021-04-11 23:58:30,780 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 146128
  total_bytes: 146128
}

2021-04-11 23:58:30,886 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 146128
  total_bytes: 146128
}

2021-04-11 23:58:30,992 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 146128
  total_bytes: 146128
}

2021-04-11 23:58:31,098 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 146128
  total_bytes: 146128
}

2021-04-11 23:58:31,205 INFO    MainThread:30675 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 1614
}
pusher_stats {
  uploaded_bytes: 146128
  total_bytes: 146128
}

2021-04-11 23:58:32,550 INFO    MainThread:30675 [wandb_run.py:_show_summary():1771] rendering summary
2021-04-11 23:58:32,550 INFO    MainThread:30675 [wandb_run.py:_show_history():1809] rendering history
2021-04-11 23:58:32,552 INFO    MainThread:30675 [wandb_run.py:_show_files():1838] logging synced files
